app/code/ClassyLlama/OrderComments/Observer/QuoteManagement.php

```diff
@@ -0,0 +1,36 @@
+<?php
+/**
+ * Quote Submit Before Observer
+ *
+ * @category ClassyLlama
+ * @package ClassyLlama_OrderComments
+ * @copyright Copyright (c) 2018 ClassyLlama
+ */
+
+namespace ClassyLlama\OrderComments\Observer;
+
+use Magento\Framework\Event\ObserverInterface;
+use Magento\Framework\Event\Observer;
+
+/**
+ * ClassyLlama\OrderComments\Observer\QuoteManagement
+ *
+ * @category ClassyLlama
+ * @package ClassyLlama_OrderComments
+ */
+class QuoteManagement implements ObserverInterface
+{
+    /**
+     * Set order comment value from quote onto order
+     *
+     * @param Observer $observer
+     *
+     * @return void
+     */
+    public function execute(Observer $observer)
+    {
+        $quote = $observer->getEvent()->getQuote();
```

Comment:
Please review Magento documentation and follow their approach if it's possible https://devdocs.magento.com/guides/v2.1/ext-best-practices/tutorials/copy-fieldsets.html